.github/workflows/docs/publish-docs.yml

```
@@ -0,0 +1,67 @@
+            - 'docs/**'
+jobs:
+    deploy-docs:
+        name: Deploy Docs to Vercel
+        needs: test-docs
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Checkout Repository
+                uses: actions/checkout@v4
+
+            -   name: Set Up Python 3.9
+                uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)